packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue

```diff
@@ -1,8 +1,37 @@
 <template>
   <div id="oc-files-sharing-sidebar" class="oc-position-relative">
-    <div class="oc-flex">
-      <h3 v-translate class="oc-text-bold oc-text-medium oc-m-rm">Members</h3>
-      <oc-contextual-helper v-if="helpersEnabled" class="oc-pl-xs" v-bind="spaceAddMemberHelp" />
+    <div class="oc-flex oc-flex-between oc-flex-center oc-flex-middle">
+      <div class="oc-flex">
```

Comment:
If you give this div a padding top and bottom of 6px (or more) each, there is no jumpiness at all anymore.